<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>529</width>
    <height>639</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>529</width>
    <height>639</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>529</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>529</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>530</width>
      <height>640</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>530</width>
      <height>640</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>530</width>
      <height>640</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="calc">
     <attribute name="title">
      <string>Calc</string>
     </attribute>
     <widget class="QPushButton" name="b_sin">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_acos">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>470</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_plus">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_minus">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_right">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>)</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_left">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>(</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_pow">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_c">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_equal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_asin">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_div">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_graph">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>234</y>
        <width>141</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_ln">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_x">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>470</y>
        <width>50</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_sqrt">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>490</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="decimals">
       <number>7</number>
      </property>
      <property name="maximum">
       <double>1000000000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="b9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_tan">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>470</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_log">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>log</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_atan">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b0">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>111</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>46</y>
        <width>450</width>
        <height>110</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>110</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QPushButton" name="b_muilti">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>350</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_mod">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>470</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_cos">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>470</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="b_dot">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="credit">
     <attribute name="title">
      <string>Credit</string>
     </attribute>
     <widget class="QTableWidget" name="credit_table">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>323</y>
        <width>510</width>
        <height>280</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>510</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>510</width>
        <height>280</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="credit_calc">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>282</y>
        <width>70</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Расчет</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>281</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="s_loan">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximum">
          <double>100000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="s_term">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>100</height>
          </size>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>600.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="s_percent">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="l_loan">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Сумма кредита(RUB)</string>
         </property>
         <property name="buddy">
          <cstring>s_loan</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_term">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Срок(Months)</string>
         </property>
         <property name="buddy">
          <cstring>s_term</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_percent">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Процентная ставка(%)</string>
         </property>
         <property name="buddy">
          <cstring>s_percent</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="mount_pay">
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Ежемесячный платеж: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="overpayment">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Переплата по кредиту:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_payout">
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Общая выплата:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>285</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="credit_radio_annu">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Аннуитетный</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="credit_radio_diff">
         <property name="text">
          <string>Дифференцированный</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="debit">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Debit</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>b_right</tabstop>
  <tabstop>b_left</tabstop>
  <tabstop>b_pow</tabstop>
  <tabstop>b_c</tabstop>
  <tabstop>b_equal</tabstop>
  <tabstop>b_asin</tabstop>
  <tabstop>b4</tabstop>
  <tabstop>b5</tabstop>
  <tabstop>b_div</tabstop>
  <tabstop>b_graph</tabstop>
  <tabstop>b_ln</tabstop>
  <tabstop>b_x</tabstop>
  <tabstop>b_sqrt</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>b9</tabstop>
  <tabstop>b6</tabstop>
  <tabstop>b_tan</tabstop>
  <tabstop>b_log</tabstop>
  <tabstop>b_atan</tabstop>
  <tabstop>b3</tabstop>
  <tabstop>b1</tabstop>
  <tabstop>b0</tabstop>
  <tabstop>b7</tabstop>
  <tabstop>b_muilti</tabstop>
  <tabstop>b2</tabstop>
  <tabstop>b8</tabstop>
  <tabstop>b_mod</tabstop>
  <tabstop>b_cos</tabstop>
  <tabstop>b_dot</tabstop>
  <tabstop>b_minus</tabstop>
  <tabstop>b_plus</tabstop>
  <tabstop>b_sin</tabstop>
  <tabstop>b_acos</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
